pkg/webhandlers/error_template.go

```diff
@@ -0,0 +1,210 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhandlers
+
+import (
+	"encoding/json"
+	"html/template"
+	"net/http"
+	"time"
+
+	"go.thethings.network/lorawan-stack/v3/pkg/errors"
+)
+
+// Data contains data to render templates.
+type Data struct {
+	ErrorTitle          string
+	ErrorMessage        string
+	ErrorID             string
+	CorrelationID       string
+	BackendErrorDetails string
+	Year                int
+	IsGenericNotFound   bool
+}
+
+const errorHTML = `
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Error - The Things Stack Enterprise</title>
```

Comment:
We're in open source here.

Can we embed a file with embedded files? https://pkg.go.dev/embed